src/current/cockroachcloud/stream-changefeed-to-snowflake-aws.md

```diff
@@ -60,89 +59,76 @@ If you have not done so already, [create a cluster](create-your-cluster.html).
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > CREATE DATABASE cdc_test;
-    ~~~
-    ~~~
-    CREATE DATABASE
+    CREATE DATABASE cdc_test;
     ~~~
 
 1. Set it as the default:
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > SET DATABASE = cdc_test;
-    ~~~
-    ~~~
-    SET
+    SET DATABASE = cdc_test;
     ~~~
 
 ## Step 4. Create tables
 
 Before you can start a changefeed, you need to create at least one table for the changefeed to target. The targeted table's rows are referred to as the "watched rows".
 
-Let's create a table called `order_alerts` to target:
+Create a table called `order_alerts` to target:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE TABLE order_alerts (
+CREATE TABLE order_alerts (
     id   INT PRIMARY KEY,
     name STRING
 );
 ~~~
-~~~
-CREATE TABLE
-~~~
 
 ## Step 5. Create an S3 bucket in the AWS Console
 
 Every change to a watched row is emitted as a record in a configurable format (i.e., `JSON` for cloud storage sinks). To configure an AWS S3 bucket as the cloud storage sink:
 
 1. Log in to your [AWS S3 Console](https://s3.console.aws.amazon.com/).
 
-1. Create an S3 bucket, called `changefeed-example`, where streaming updates from the watched tables will be collected.
+1. Create an S3 bucket where streaming updates from the watched tables will be collected.
 
-    The name of the S3 bucket is needed when you [create your changefeed](#step-6-create-an-enterprise-changefeed). Be sure to have a set of IAM credentials with write access on the S3 bucket that will be used during [changefeed setup](#step-6-create-an-enterprise-changefeed).
+    You will need the name of the S3 bucket when you [create your changefeed](#step-6-create-an-enterprise-changefeed). Ensure you have a set of IAM credentials with write access on the S3 bucket that you will use during [changefeed setup](#step-6-create-an-enterprise-changefeed).
```

Comment:
can call out this information in a note

Reply:
I think this is really part of the workflow — where possible I do not want to overuse notes/warnings/tips as it then diminishes there usefulness throughout the docs.

Reply:
Okay, makes sense to me!